### ISSUE TITLE
[popup] Should we rename showPopup/hidePopup to showPopUp/hidePopUp?

### ISSUE THREAD

**Issue by mfreed7** [openui/open-ui#549]
Per the [recent resolution](https://github.com/openui/open-ui/issues/546#issuecomment-1158190204), we've decided to call this API the "pop up" API, and rename the IDL property accessor to `popUp` (with a capital `U`). Should we therefore also rename the rest of the JS-accessible parts of the API to `popUp`? E.g.:

```javascript
myPopUp.showPopUp();
myPopUp.hidePopUp();
myButton.togglePopUp = 'idref';
myButton.showPopUp = 'idref';
myButton.hidePopUp = 'idref';
```

? It feels a bit cumbersome to type, as I'm typing the above. But it's definitely more consistent if we've decided to use `myPopUp.popUp = "auto"`. So it feels like we should make the above changes.

@domenic thoughts?

**Comment by domenic** [comment]
Yes, this is a necessary part of the previous resolution. There is no such thing as a "popup", if you go that route.

**Comment by mfreed7** [comment]
> Yes, this is a necessary part of the previous resolution. There is no such thing as a "popup", if you go that route.

Yeah ok, makes sense to me. I'm going to just assume that, close this issue, and implement it in the Chromium prototype.
